docs/public/content/guide.md

```diff
@@ -44,7 +44,7 @@ npx elm-spa server
 So far, we've used [npx](https://www.npmjs.com/package/npx) so we don't need to install __elm-spa__ directly. If you'd like to run commands from the terminal, without the `npx` prefix, you can install __elm-spa__ like this:
 
 ```terminal
-npm install -g elm-spa@next
```

Comment:
I'm planning on changing this to `next` when the website goes live, so it's incorrect for now– but should be `next` soon.

I'm planning for all website URLs to be `https://next.elm-spa.dev` and all installs should be `elm-spa@next`